Packages/com.unity.inputsystem/InputSystem/InputSettings.cs

```diff
@@ -301,6 +302,35 @@ public float defaultButtonPressPoint
             }
         }
 
+        /// <summary>
+        /// The percentage of <see cref="defaultButtonPressPoint"/> at which a button that was pressed
+        /// is considered released again.
+        /// </summary>
+        /// <remarks>
+        /// This setting helps avoid flickering around the button press point by introducing something akin to a
+        /// "dead zone" below <see cref="defaultButtonPressPoint"/>. Once a button has been pressed to a magnitude
+        /// of at least <see cref="defaultButtonPressPoint"/>, it is considered pressed and keeps being considered pressed
+        /// until its magnitude falls back to a value of or below <see cref="buttonReleaseThreshold"/> percent of
+        /// <see cref="defaultButtonPressPoint"/>.
+        ///
+        /// This is a percentage rather than a fixed value so it allows computing release
+        /// points even when the press point has been customized. If, for example, a <see cref="Interactions.PressInteraction"/>
+        /// sets a custom <see cref="Interactions.PressInteraction.pressPoint"/>, the the respective release point
```

Comment:
the the

Reply:
Good catch. Found a few more instances of this.